### PR TITLE
docdb has tags

### PR DESCRIPTION
**Pull request by muzzamilinovaqo**

### CODE REVIEW

**Comment by fatima99s** [comment]
resolve the confilts

File: plugins/aws/documentDB/docDbHasTags.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    link: 'https://docs.aws.amazon.com/documentdb/latest/developerguide/tagging.html',
+    apis: ['DocDB:describeDBClusters', 'DocDB:listTagsForResource'],
+    realtime_triggers: ['docdb:CreateDBCluster','docdb:CreateDBInstance','docdb:DeleteDBCluster', 'docdb:ModifyDBCluster'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+
```
Comment (fatima99s) [inline plugins/aws/documentDB/docDbHasTags.js:21]:
```suggestion
```

File: plugins/aws/documentDB/docDbHasTags.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                helpers.addResult(results, 3,
+                    `Unable to list DocumentDB clusters: ${helpers.addError(describeDBClusters)}`, region);
+                return rcb();
+            }
+
+            if (!describeDBClusters.data.length) {
+                helpers.addResult(results, 0,
+                    'No DocumentDB clusters found', region);
+                return rcb();
+            }
```
Comment (fatima99s) [inline plugins/aws/documentDB/docDbHasTags.js:38]:
```suggestion
            }
            
```

File: plugins/aws/documentDB/docDbHasTags.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            for (let cluster of describeDBClusters.data){
```
Comment (fatima99s) [inline plugins/aws/documentDB/docDbHasTags.js:39]:
```suggestion
            for (let cluster of describeDBClusters.data) {
```

**Review by fatima99s — CHANGES_REQUESTED** [review]

File: helpers/aws/api_multipart.js [hunk 4]
```diff
@@ -1406,6 +1406,14 @@ var postcalls = [
                 override: true
             }
         },
+        DocDB: {
```
Comment (alphadev4) [inline helpers/aws/api_multipart.js:1409]: cant we use resourceGroupTaggingApi call?

File: plugins/aws/documentDB/docDbHasTags.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            for (let cluster of describeDBClusters.data) {
+                if (!cluster.DBClusterArn) continue;
+               
+                let resource = cluster.DBClusterArn;
+
+                let getTags = helpers.addSource(cache, source,
+                    ['docdb', 'listTagsForResource', region, resource]);
+
+                if (!getTags || !getTags.data || getTags.err) {
+                    helpers.addResult(results, 3, `Unable to get tags information for doc db cluster: ${helpers.addError(getTags)}`, region, resource);
```
Comment (alphadev4) [inline plugins/aws/documentDB/docDbHasTags.js:48]:
```suggestion
                    helpers.addResult(results, 3, `Unable to get tags for DocumentDB cluster: ${helpers.addError(getTags)}`, region, resource);
```

File: plugins/aws/documentDB/docDbHasTags.spec.js [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+                {
+                    DBClusterArn: 'arn:aws:rds:us-east-1:000011112222:cluster:test-cluster'
+                }
+            ];
+
+            const callback = (err, results) => {
+                expect(results.length).to.equal(1);
+                expect(results[0].status).to.equal(3);
+                expect(results[0].region).to.equal('us-east-1');
+                expect(results[0].message).to.include('Unable to get tags information for doc db cluster');
```
Comment (alphadev4) [inline plugins/aws/documentDB/docDbHasTags.spec.js:52]:
```suggestion
                expect(results[0].message).to.include('Unable to get tags for DocumentDB cluster:');
```

**Review by alphadev4 — APPROVED** [review]

**Pull request merged**